Subject: [PATCH 14/18] Add description of sadump-related formats in manual page

This is a backport of

    commit c28ac15797b643f7d7203a1c375aa200ac5d684a
    Author: HATAYAMA Daisuke <d.hatayama@jp.fujitsu.com>
    Date:   Fri Dec 16 11:02:20 2011 +0900
    
    [PATCH v2 14/14] Add description of sadump-related formats in manual page
    
    Signed-off-by: HATAYAMA Daisuke <d.hatayama@jp.fujitsu.com>
---
diff -Nrup kexec-tools-2.0.0.old/makedumpfile-1.3.5/makedumpfile.8 kexec-tools-2.0.0/makedumpfile-1.3.5/makedumpfile.8
--- kexec-tools-2.0.0.old/makedumpfile-1.3.5/makedumpfile.8	2012-04-05 07:21:03.126497571 -0400
+++ kexec-tools-2.0.0/makedumpfile-1.3.5/makedumpfile.8	2012-04-05 08:02:36.480508030 -0400
@@ -20,6 +20,8 @@ makedumpfile \- make a small dumpfile of
 .br
 \fBmakedumpfile\fR \-\-dump-dmesg [\-x \fIVMLINUX\fR|\-i \fIVMCOREINFO\fR] \fIVMCORE\fR \fILOGFILE\fR
 .br
+\fBmakedumpfile\fR    [\fIOPTION\fR] \-x \fIVMLINUX\fR \-\-diskset=\fIVMCORE1\fR \-\-diskset=\fIVMCORE2\fR [\-\-diskset=\fIVMCORE3\fR ..] \fIDUMPFILE\fR
+.br
 .B makedumpfile
 \-h
 .br
@@ -104,6 +106,14 @@ one of dump_level 3 like the following:
 # makedumpfile \-c \-d 3 /proc/vmcore dumpfile.1
 .br
 # makedumpfile \-c \-d 31 dumpfile.1 dumpfile.2
+.PP
+makedumpfile can read \fIVMCORE\fR(s) in three kinds of sadump
+formats: single partition format, diskset format and media backup
+format, and can convert each of them into kdump-compressed format with
+filtering and compression processing. Note that for \fIVMCORE\fR(s)
+created by sadump, you always need to pass \fIVMLINUX\fR with -x
+option. Also, to pass multiple \fIVMCORE\fRs created on diskset
+configuration, you need to use --diskset option.
 
 .PP
 .SH OPTIONS
@@ -460,6 +470,18 @@ it is necessary to specfiy [\-x \fIVMLIN
 .br
 
 .TP
+\fB\-\-diskset=VMCORE\fR
+Specify multiple \fIVMCORE\fRs created on sadump diskset configuration
+the same number of times as the number of \fIVMCORE\fRs in increasing
+order from left to right.  \fIVMCORE\fRs are assembled into a single
+\fIDUMPFILE.
+
+.br
+.B Example:
+.br
+# makedumpfile \-x vmlinux \-\-diskset=vmcore1 \-\-diskset=vmcore2 dumpfile
+
+.TP
 \fB\-D\fR
 Print debugging message.
 
